### PR TITLE
Add a function to create disk on local or remote

### PR DESCRIPTION
**Pull request by Yingshun**
This is a wrapper method to create disk on local/remote.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

### CODE REVIEW

**Comment by Yingshun** [comment]
@dzhengfy Could you please help review? thx!

**Comment by Yingshun** [comment]
**Test result:**
` (1/1) type_specific.io-github-autotest-libvirt.virsh.migrate_over_unix.positive_testing.with_copy_storage.default.multi_disks.p2p_live_migration.without_postcopy: PASS (111.91 s)`

File: virttest/utils_libvirt/libvirt_misc.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"""
```
Comment (dzhengfy) [inline virttest/utils_libvirt/libvirt_misc.py:1]: Regarding the python file name, is it better to use libvirt_disk.py ?

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: virttest/utils_libvirt/libvirt_misc.py (hunk 1 above)
Comment (Yingshun) [reply virttest/utils_libvirt/libvirt_misc.py]: Thanks, updated.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Comment by Yingshun** [comment]
@chloerh Could you please help review this pr? Thank you!

File: virttest/utils_libvirt/libvirt_disk.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    :return: The path of disk
+    :raise: TestError if the disk can't be created
+    """
+
+    if session:
+        if disk_type == "file":
+            disk_cmd = ("qemu-img create -f %s %s %s %s"
+                        % (disk_format, extra, path, size))
+        else:
+            # TODO: Add implementation for other types
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_disk.py:31]: :joy:

**Review by chloerh — APPROVED** [review]

**Pull request merged**